edx_exams/apps/core/email.py

```diff
@@ -44,7 +47,6 @@ def send_attempt_status_email(attempt, escalation_email=None):
         contact_url = f'{settings.LMS_ROOT_URL}/support/contact_us'
         contact_url_text = contact_url
 
-    email_subject = f'Proctored exam {exam.exam_name} for user {attempt.user.username}'
```

Comment:
Why can we not leave the exam name in?

Reply:
Actually we have discussed subject line with UX team and they suggested us that we can remove the exam name from subject but if you still think we should get it back we can then discuss with UX again. Thanks